[New post] How to build GenAI application using Amazon Bedrock with AWS PrivateLink to protect your data privacy (#961)
build-on-aws/content: concepts/build-GenAI-app-Bedrock-privateLink/index.md

> ---

Reviewer: Hello,

The blog has good structure and a strong intro that highlights the reasons why someone would need to read the blog. I made edits on grammar, titles and left a few notes where I got confused. Please see below for more information. 

Overall, you have a good blog here. Well done on a first draft!

> title: "How to build GenAI application using Amazon Bedrock with AWS PrivateLink to protect your data privacy"  

Reviewer: ```suggestion
title: "Build GenAI Applications Using Amazon Bedrock With AWS PrivateLink To Protect Your Data Privacy"  
```

> description: "This post is about **what** is **Private Link**, **Why** to use Private Link to protect Data Privacy, and **How** to build GenAI application using Amazon Bedrock with Private Link."

Reviewer: The descriptions need to be 60 characters or fewer. They can't repeat the messaging in the title. They should focus on what solution the reader gets from reading the piece. I wrote an example below but feel free to rewrite.
 
```suggestion
description: "Manage and protect data security and compliance standards."
```

tags:  
  - Amazon Bedrock
  - GenAI
  - Private Link
  - Data Privacy
authorGithubAlias: Xiaoyuao988
authorName: Xiaoyu Xing
date: 2023-12-20
---

Most would likely agree that **Data Privacy** is an important issue that cannot be overlooked when it comes to data analytics and machine learning. **Generative AI** is making waves, but without proper security control, Generative AI tools could be susceptible to data theft. This could lead to unauthorized access or disclosure of sensitive information, potentially leading to privacy violations and possible misuse of both sensitive personal data or intellectual property.

[Amazon Bedrock](https://aws.amazon.com/bedrock/security-compliance/), a fully managed service, allows you to build generative AI applications that support data security and compliance standards, including **GDPR** and **HIPAA**.

> With **Amazon Bedrock**, ***(not limited to)***

Reviewer: Please consider rewriting this sentence. It's incomplete. What are you trying to say here? 

```suggestion
With **Amazon Bedrock**, ***(not limited to)***
```

Author: What I mean is that I haven't listed all the benefits.
Re-write by removing "***(not limited to)***", and adding sentence: "As you can see, Amazon Bedrock provides you comprehensive security and compliance capabilities. **Here I just list some of them**."

* Your **content** (e.g. prompt data, fine-tuning data, vector store data with RAG (***Retrieval Augmented Generation***) etc.) is not shared with 3rd party model providers (including AWS).
* You can use **AWS PrivateLink** to establish private connectivity between the **Foundation Models (FM)** and on-premises networks or your **AWS VPC (Virtual Private Cloud)**, without exposing your traffic to the **public internet**. (***This blog post's focus***)
* Your data is always **encrypted in transit** (***TLS1.2***) and **at rest**. You can use your own keys or [AWS KMS (Key Management Service)](https://aws.amazon.com/kms/) keys to encrypt the data.

> * You can encrypt and store **customized (fine-tuned) models** with KMS key that either managed by AWS or by yourself.

Reviewer: ```suggestion
* You can encrypt and store **customized (fine-tuned) models** with a KMS key that's either managed by AWS or by yourself.
```

  You can further configure VPC for fine-tuning jobs with Bedrock, which prevent the **training data** from being accessible over the internet.

> * **Potential misuse** is prevented by [Bedrock abuse detection](https://docs.aws.amazon.com/bedrock/latest/userguide/abuse-detection.html). This feature **automatically** ***(no human review or access to user inputs or model outputs)*** identifies and mitigates potential violations of AWS's [Acceptable Use Policy (AUP)](https://aws.amazon.com/aup/) and [Responsible AI Policy](https://aws.amazon.com/machine-learning/responsible-ai/policy/), or a 3rd party model provider's AUP.

Reviewer: ```suggestion
* **Potential misuse** is prevented by [Bedrock abuse detection](https://docs.aws.amazon.com/bedrock/latest/userguide/abuse-detection.html). This feature **automatically** ***(no human review or access to user inputs or model outputs)*** identifies and mitigates potential violations of AWS's [Acceptable Use Policy (AUP)](https://aws.amazon.com/aup/) and [Responsible AI Policy](https://aws.amazon.com/machine-learning/responsible-ai/policy/), or a third party model provider's AUP.
```
